.github/workflows/pull_request.yml

```diff
@@ -15,3 +15,9 @@ jobs:
         with:
           branchName: ${{ github.base_ref }}
           token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: boa-dev/criterion-compare-action@v3
+        with:
+          branchName: ${{ github.base_ref }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          defaultFeatures: false
```

Comment:
This option doesn't seem to be adding ``--no-default-features`` like it should :/


idk javascript: https://github.com/boa-dev/criterion-compare-action/blob/59f4d964c5f19d7f13d36f5c0944b18ce6652cb0/main.js#L37

Reply:
Maybe I fixed it, i'll try out my test version here...

https://github.com/boa-dev/criterion-compare-action/pull/117

Reply:
Didn't fix it, guess I won't become a js dev anytime soon 😆